src/components/platforms.vue

```diff
@@ -172,6 +172,15 @@ const companies = [
     automationPlusAuditCost: '$22k',
     integrationCount: null,
     note: '<li>AI everything</li><li>Also handles the audit.</li>' },
+  {
+    link: 'https://lowerplane.com/',
+    name: 'LowerPlane',
+    automationPlatformCost: '$5k',
+    licensed: 'PARTNERS',
+    auditCost: '$5k+',
```

Comment:
Website actually says 8k+

```suggestion
    auditCost: '$8k+',
```